Add Control for BSI SYS 1.6

sluetze opened the pull request:
#### Description:

adding the review remarks:
1. introduce versioning for bsi profile (ocp4, ocp4-node and rhoccs)
2. utilize control file for rhocs profile
3. add missing references


#### Rationale:

sys.1.6 (containerization) is the second part of the relevant building blocks of BSI for container platforms.

#### Review Hints:

this is to be merged after https://github.com/ComplianceAsCode/content/pull/11394

sluetze commented on controls/bsi_sys_1_6.yml:
```diff
@@ -0,0 +1,412 @@
+# i.e. there are multiple sentences, some including a RFC2119 keyword
+# Since we must increase granularity to create a precise control,
+# we number each sentence with a RFC2119 keyword as a section, grouping sentences, which are logically connected.
+# we number inline in brackets, so the lookup is easy
+# we reference these numbers in comments over each rule or group of rules
```
Yes, since we only setup the control file for sys.1.6 and are currently prioritizing app.4.4 there is nothing like that currently in this pr.

nevertheless there is an example in another PR right here: https://github.com/ComplianceAsCode/content/blob/2439d34019425c375c0bb7689649f61df9ef6845/controls/bsi_app_4_4.yml#L70-L100

We might overload the control file with our approach so feel free to comment :)

yuumasato replied:
Thank you for the example.

> We might overload the control file with our approach so feel free to comment :)

This seems fine for now. But I see how this could lengthen the control file.

Regardless, this reminds me that control files can be split into multiple files in a directory, see for example the cis_ocp4 [file](https://github.com/ComplianceAsCode/content/blob/master/controls/cis_ocp_1_4_0.yml) and [directory](https://github.com/ComplianceAsCode/content/tree/master/controls/cis_ocp_1_4_0).

yuumasato commented:
@sluetze I see that one commit for APP.4.4.A4 was added.
Was that intentional?

sluetze commented:
> @sluetze I see that one commit for APP.4.4.A4 was added. Was that intentional?

@yuumasato yes this is intentional. Since this PR addresses the review marks of https://github.com/ComplianceAsCode/content/pull/11342, i also fixed the issue, that the rules are splitted across multiple profiles. before this PR the rules of APP.4.4 were in the rhcos bsi profile, which made them invisible in the ocp bsi profile. Now we have the rules also in the ocp profile while keeping the variables in the rhcos profile because putting them in the ocp profile results in errors.

Also the references were not put in place in the last PR so i fixed that.

teacup-on-rockingchair reviewed (approved):
LGTM :bow:

yuumasato reviewed (approved):
Thanks for the clarifications

dodys reviewed (approved):
lgtm, thanks

sluetze commented:
rebased to fix merge conflicts.

yuumasato commented:
There are no maintainers in Oracle group, overriding and merging, :)

yuumasato commented:
Err, re-running the testing-farm jobs that failed.

yuumasato commented:
Thank you @sluetze!
